rita_client/src/operator_update/mod.rs

```diff
@@ -70,6 +71,11 @@ lazy_static! {
         Arc::new(RwLock::new(UptimeStruct::new()));
     static ref OPERATOR_UPDATE: Arc<RwLock<OperatorUpdate>> =
         Arc::new(RwLock::new(OperatorUpdate::default()));
+    pub static ref CONTACT_INFO: Arc<RwLock<ContactDetails>> =
```

Comment:
this needs a doc-comment describing the workflow, we set the contact info when the user registers to an exit or otherwise sets their contract info. At that point it's sent up to ops tools and then pulled back down from ops tools on next run.